app/Platform/Actions/UpdateGamePlayerGames.php

```diff
@@ -18,7 +18,10 @@ public function execute(Game $game): void
     {
         // don't do this without a proper queue
         if (config('queue.default') === 'sync') {
-            return;
+            // do allow this to cascade in the unit tests
+            if (!app()->environment('testing')) {
+                return;
+            }
```

Comment:
it should cascade for tests and returns if it's not testing which makes sense but confused me because testing is always run with sync driver. the only other variant that allows to run sync is when set locally.
could this be rephrased positively to read for `local` specifically or would this be more confusing?
what we want to prevent here specifically is that when running with a huge dataset locally that the sync queue would run for a very long time which is not a problem with tests.
`if (app()->environment('local')) {`

Reply:
the more i think about it the more it makes sense how you wrote it - feel free to discard :)

Reply:
it could be inlined though:
```php
// don't do this without a proper queue and not testing
if (config('queue.default') === 'sync' && !app()->environment('testing')) {
    return;
}
```